app/utils/answersHelpers/consignment/HouseConsignmentAnswersHelper.scala

```diff
@@ -38,15 +39,13 @@ class HouseConsignmentAnswersHelper(
 )(implicit messages: Messages)
     extends AnswersHelper(userAnswers) {
 
-  def safetyAndSecurityDetails: SummaryListRow =
-    buildRowWithNoChangeLink(
-      prefix = "houseConsignment.securityIndicator",
-      answer = formatAsYesOrNo(
-        userAnswers.ie043Data.Consignment
-          .flatMap(_.HouseConsignment(houseConsignmentIndex.position).securityIndicatorFromExportDeclaration)
-          .contains("1")
-      )
-    )
+  def safetyAndSecurityDetails: Option[SummaryListRow] = getAnswerAndBuildRow[SecurityType](
+    page = SecurityIndicatorFromExportDeclarationPage(houseConsignmentIndex),
+    formatAnswer = x => formatAsText(s"${x.code} - ${x.description}"),
```

Comment:
In the ConsignmentAnswersHelper we just cast this to a string (which only displays the description). Worth being consistent. Check with Gavin if he'd prefer `description` or `code - description`